sdk/python/feast/feature_view.py

```diff
@@ -379,3 +388,23 @@ def infer_features_from_batch_source(self, config: RepoConfig):
                     "FeatureView",
                     f"Could not infer Features for the FeatureView named {self.name}.",
                 )
+
+    def set_projection(self, feature_view_projection: FeatureViewProjection) -> None:
+        """
+        Setter for the projection object held by this FeatureView. Performs checks to ensure
```

Comment:
Nitpick: These comments assume the user knows exactly what a projection is. I think it's important to define what a projection is.